lib/Sema/DerivedConformances.h

```diff
@@ -296,10 +302,7 @@ class DerivedConformance {
   /// Derive a Decodable requirement for a struct type.
   ///
   /// \returns the derived member, which will also be added to the type.
-  ValueDecl *deriveDecodable(ValueDecl *requirement);  
-
-  /// Whether we can derive the given DistributedActor requirement in the given context.
-  static bool canDeriveDistributedActor(NominalTypeDecl *nominal, DeclContext *dc);
```

Comment:
there is `ValueDecl *deriveDistributedActor(ValueDecl *requirement);` just below here which is the one we use